docker/airflow/entrypoint.sh

```diff
@@ -38,20 +38,27 @@ sleep 0.1;  # The $COLUMNS variable takes a moment to populate
 header "MODIFYING ENVIRONMENT"
 # Loop through environment variables, relying on naming conventions.
 # Bash loops with pipes occur in a subprocess, so we need to do some special
-# subprocess manipulation via <(...) syntax in order to allow the `export` calls
+# subprocess manipulation via <(...) syntax to allow the `export` calls
 # to propagate to the outer shell.
 # See: https://unix.stackexchange.com/a/402752
-while IFS="="; read var_name var_value; do
-    echo "Variable Name : $var_name"
-    echo "    Original Value: $var_value"
-    # call python to get the new value
-    url_encoded=`python -c"from urllib.parse import quote_plus; import sys; print(quote_plus(sys.argv[1]))" $var_value`
+while read var_string; do
+    # get the variable name
+    var_name=`expr "$var_string" : '^\([A-Z_]*\)'`
+    echo "Variable Name: $var_name"
+    # get the old value
+    old_value=`expr "$var_string" : '^[A-Z_]*=\(http.*\)$'`
+    echo "    Old Value: $old_value"
+    # if http_clause starts with http, then replace http with https
+    url_encoded="${old_value/"http:"/"https:"}"
```

Comment:
Hmm, I think we want to _keep_ this as `http`, since we're not likely to have certs set up for local development.